assets/media/alt-tag-enhancements.js

```diff
@@ -1,16 +1,16 @@
-import $ from 'jquery'
 import forEach from 'lodash/forEach'
-import get from 'lodash/get'
 import './alt-tag.scss'
-const MEDIA_IMAGE_TYPE = 'image';
+
+const $ = window.jQuery
```

Comment:
maybe put this in a jquery.js file so we can do

import $ from './jquery'

Reply:
Agreed in that way it will look like cleaner dep :v